chef/automate
# Data feed UI changes/ #5414 & #5415

### :nut_and_bolt: Description: What code changed, and why?



### :chains: Related Resources

### :+1: Definition of Done
Automate data-feed page should look like the image below.
<img width="1779" alt="Screenshot 2021-08-18 at 3 41 38 PM" src="https://user-images.githubusercontent.com/13744874/129888532-2143d31e-2002-4466-bf02-e80c112e7871.png">

### :athletic_shoe: How to Build and Test the Change
1. Go to settings > click feat on tab to enable feature flag (on Chef Automate Data Feed option) and reload page - Data feed option should come below Notification tab
2. Click on data-feed 
3. Empty data-feed should look like the image above
To test:
1. Go to e2e/cypress folder in automate
2. run npm run cypress:local

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
<img width="1779" alt="Screenshot 2021-08-18 at 3 41 38 PM" src="https://user-images.githubusercontent.com/13744874/129882595-8734c67b-e131-48f3-8e53-944d6491ec72.png">
